xchange-lgo/src/main/java/org/knowm/xchange/lgo/dto/order/LgoPlaceOrder.java

```diff
@@ -0,0 +1,28 @@
+package org.knowm.xchange.lgo.dto.order;
+
+import java.time.Instant;
+
+public abstract class LgoPlaceOrder {
+
+  private long reference;
```

Comment:
setReference is not user anywhere, lets make it immutable

Reply:
👍